Sources/PrivoSDK/api/Rest.swift

```diff
@@ -14,12 +14,16 @@ struct URLComponentConstants: ExpressibleByStringLiteral, RawRepresentable {
 }
 
 extension URLComponentConstants {
-    static let analytic: URLComponentConstants = "metrics"
```

Comment:
why do we need this enum? It's just string path's

Reply:
updated